### ISSUE TITLE
Build as a .dylib?

### ISSUE THREAD

**Issue by hiburn8** [remirobert/Dotzu#40]
I'm a security consultant and regularly test pre-build iOS binaries for security issues. This looks like another great tool to add to the kit-bag for testing the networking of apps... although, i cannot for the life of me figure out how to build this as a dynamic library. I know XCode no longer has the options for dynamic frameworks enabled so this would require some command-line voodoo. I've looked at modifying this https://github.com/danylokostyshyn/flex-dylib-build-script
But this no longer works anyway so i'm a bit lost. Any help would be appreciated. Thanks.

**Comment by remirobert** [comment]
It's very interesting, that's a use case I haven't think of.
I don't know if that's possible, especially with swift.

I will definitely check that. During that time if you find any resource that can help me to figure it out, reply to that issue.

**Comment by hiburn8** [comment]
Thanks for the response, i've just seen it. Still not having much luck, although i would still be keen to get it working.

**Comment by johndpope** [comment]
@hiburn8  -  you may want to probably checkout revealapp.com
there's a lot of hacks to inject their dylib into prebuilt apps.
https://gist.github.com/search?utf8=%E2%9C%93&q=reveal+dylib&ref=searchresults

**Comment by liman123** [comment]
Hi, I released [DebugMan](https://github.com/liman123/DebugMan), it is forked from [Dotzu](https://github.com/remirobert/Dotzu).
Welcome to use! 😃 thanks!
@hiburn8

**Comment by hiburn8** [comment]
True. But reveal doesnt have on-device inspection, it relies on a mac app to actually view stuff. And the main thing i like about Dotzu is the networking inspector, which reveal cant do. My understanding is Reveal is just for looking at app view hierarchies, mostly for UI development :(
